src/conduits/RwaMultiSwapOutputConduit.sol

```diff
@@ -0,0 +1,466 @@
+     * @param usr The user address.
+     */
+    event Hope(address indexed usr);
+    /**
+     * @notice `usr` operator access was revoked.
+     * @param usr The user address.
+     */
+    event Nope(address indexed usr);
+    /**
+     * @notice `usr` was granted push access.
+     * @param usr The user address.
+     */
+    event Mate(address indexed usr);
+    /**
+     * @notice `usr` push access was revoked.
+     * @param usr The user address.
+     */
+    event Hate(address indexed usr);
+    /**
+     * @notice `who` address whitelisted for pick.
+     * @param who The user address.
+     */
+    event Kiss(address indexed who);
+    /**
+     * @notice `who` address was removed from whitelist.
+     * @param who The user address.
+     */
+    event Diss(address indexed who);
+    /**
+     * @notice `psm` address whitelisted for hook.
+     * @param psm The PSM address.
+     */
+    event Clap(address indexed psm);
+    /**
+     * @notice `psm` address was removed from whitelist.
+     * @param psm The user address.
+     */
+    event Slap(address indexed psm);
+    /**
+     * @notice `psm` address was choosen.
+     * @param psm The PSM address.
+     */
+    event Hook(address indexed psm);
+    /**
+     * @notice `who` address was picked as the recipient.
+     * @param who The user address.
+     */
+    event Pick(address indexed who);
+    /**
+     * @notice `amt` amount of GEM was pushed to the recipient `to` using `psm`.
+     * @param psm PSM address used for swap.
+     * @param gem GEM token address used.
+     * @param to Destination address for GEM.
+     * @param amt The amount of GEM.
+     */
+    event Push(address indexed psm, address indexed gem, address indexed to, uint256 amt);
+    /**
+     * @notice A contract parameter was updated.
+     * @param what The changed parameter name. Currently the supported values are: "quitTo", "psm".
+     * @param data The new value of the parameter.
+     */
+    event File(bytes32 indexed what, address data);
+    /**
+     * @notice The conduit outstanding DAI balance was flushed out to `quitTo` address.
+     * @param quitTo The quitTo address.
+     * @param wad The amount of DAI flushed out.
+     */
+    event Quit(address indexed quitTo, uint256 wad);
+    /**
+     * @notice `amt` outstanding `token` balance was flushed out to `usr`.
+     * @param token The token address.
+     * @param usr The destination address.
+     * @param amt The amount of `token` flushed out.
+     */
+    event Yank(address indexed token, address indexed usr, uint256 amt);
+
+    modifier auth() {
+        require(wards[msg.sender] == 1, "RwaMultiSwapOutputConduit/not-authorized");
+        _;
+    }
+
+    modifier onlyMate() {
+        require(may[msg.sender] == 1, "RwaMultiSwapOutputConduit/not-mate");
+        _;
+    }
+
+    modifier onlyOperator() {
+        require(can[msg.sender] == 1, "RwaMultiSwapOutputConduit/not-operator");
+        _;
+    }
```

Comment:
Nit-pick: `onlyOperator` is used before `onlyMate`, so I believe it should be declared first.
```suggestion

    modifier onlyOperator() {
        require(can[msg.sender] == 1, "RwaMultiSwapOutputConduit/not-operator");
        _;
    }
    
    modifier onlyMate() {
        require(may[msg.sender] == 1, "RwaMultiSwapOutputConduit/not-mate");
        _;
    }
```